modules/widgets/top-posts.php

```diff
@@ -488,12 +488,14 @@ function get_by_views( $count, $args ) {
 			$days = 2;
 		}
 
-		$post_view_posts = stats_get_csv( 'postviews', array( 'days' => absint( $days ), 'limit' => 11 ) );
-		if ( ! $post_view_posts ) {
+		$post_view_posts = stats_get_from_restapi( array(), 'top-posts?max=10&summarize=1&num=' . absint( $days ) );
```

Comment:
Exactly, if I added them as array in that first argument they were filtered out so I added them as query parameters.

There's no explanation about the 11 in [the commit that introduced it](https://github.com/Automattic/jetpack/commit/87f03f54e5e02ac1264a980745cd1dd02856042d#diff-dd11499ff5fdaf0fca3a7ec45dada777) and the widget says it will returns 10 at most so it makes sense to set it at 10. Please let me now if I'm missing something here.